mithril-common/src/entities/mithril_stake_distribution.rs

```diff
@@ -70,11 +70,11 @@ mod tests {
 
     use super::*;
 
-    const EXPECTED_HASH: &str = "47675a6fad57040b194655b103bc0439ff9d6920a7ba86bd53756db7ce2952f5";
+    const EXPECTED_HASH: &str = "c5c1ff02e37c751329e3db7625c77fa2a24e86b2a75422c54f1b9f9232374d6f";
```

Comment:
That's weird, the tests `test_compute_hash` and `test_independence_protocol_parameters` are failing on my local environment.
The expected hash is: `2d160ff149e85b85a8c66f6f2355bddd799c808a275afdadfb798b2bcd5a457a`

Reply:
Thanks

Reply:
I didn't have the info in mind anymore, sorry 😓